server/src/main/java/org/opensearch/discovery/PeerFinder.java

```diff
@@ -84,14 +84,23 @@ public abstract class PeerFinder {
         Setting.Property.NodeScope
     );
 
+    // the time between attempts to find all peers when node is in decommissioned state, default set to 3 minutes
+    public static final Setting<TimeValue> DISCOVERY_FIND_PEERS_INTERVAL_DURING_DECOMMISSION_SETTING = Setting.timeSetting(
+        "discovery.find_peers_interval_during_decommission",
+        TimeValue.timeValueMinutes(3L),
```

Comment:
Let's keep it to 120s as default. Lets follow it up with a exponential back off strategy with plateau

Reply:
Done. Have opened an issue to track the backoff strategy changes